src/app/component/user/components/profile/profile-dashboard/one-habit/one-habit.component.html

```diff
@@ -12,6 +12,7 @@
         <img class="first-photo" [src]="firstFriend" alt="First friend photo">
         <img class="second-photo" [src]="secondFriend" alt="Second friend photo">
       </div>
+      <div class="edit" (click)="goHabitMore()"></div>
```

Comment:
can you make a more appropriate name for the function? If it edits something maybe consider a name like  'goEditHabit'

Reply:
changed

Reply:
### changed